app/controllers/api/databases.php

```diff
@@ -2973,7 +2974,11 @@ function updateAttribute(
     ->inject('dbForProject')
     ->inject('mode')
     ->action(function (string $databaseId, string $collectionId, array $queries, Response $response, Database $dbForProject, string $mode) {
-        $database = Authorization::skip(fn() => $dbForProject->getDocument('databases', $databaseId));
+        try {
+            $database = Authorization::skip(fn() => $dbForProject->getDocument('databases', $databaseId));
+        } catch (DatabaseException $e) {
```

Comment:
Shouldn't we just get an empty document here instead of a DatabaseException?

Reply:
Because the 'databases' collection does not exists, an Exception is being thrown.
We can see in the error  
$dbForProject is the console , that database does not exist in the console DB , if possible it should not even get to this route

Reply:
projectId=console

Reply:
Don't we have the same thing in a gazillion places? @abnegate a different approach would be to disable databases, functions and other services the console don't use in the hardcoded console project we have.

Reply:
True that makes a lot more sense! I think it will fix other cases we've seen with similar issues @fogelito